### PR TITLE
Functional tests - Add some tests in 'Design > Theme & logo > Pages configuration'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Add some tests in 'Design > Theme & logo > Pages configuration'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Nightly => https://github.com/nesrineabdmouleh/testing_pr/actions/runs/4938350131
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts [hunk 1]
```diff
@@ -42,9 +51,54 @@ export default class themeAndLogoBasePage extends BOBasePage {
...
   }
 
   /* Methods */
+  /**
+   * Get growl message content
+   * @param page {Page} Browser tab
+   * @param timeout {number} Timeout to wait for the selector
+   * @return {Promise<string|null>}
+   */
+  getGrowlMessageContent(page: Page, timeout: number = 10000): Promise<string | null> {
```
Comment (Progi1984) [inline tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts]:
```suggestion
  async getGrowlMessageContent(page: Page, timeout: number = 10000): Promise<string | null> {
```

File: tests/UI/pages/BO/design/themeAndLogo/pagesConfiguration/index.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import themeAndLogoBasePage from '@pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage';
+import {Page} from 'playwright';
+import ModuleData from '@data/faker/module';
```
Comment (Progi1984) [inline tests/UI/pages/BO/design/themeAndLogo/pagesConfiguration/index.ts]:
```suggestion
// Import pages
import themeAndLogoBasePage from '@pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage';

// Import data
import ModuleData from '@data/faker/module';

import {Page} from 'playwright';
```

**Review by Progi1984 — COMMENTED** [review]
Small feedbacks ;)

File: tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts]: Done ;)

File: tests/UI/pages/BO/design/themeAndLogo/pagesConfiguration/index.ts (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/design/themeAndLogo/pagesConfiguration/index.ts]: Done ;)

**Review by Progi1984 — DISMISSED** [review]

**Review by boubkerbribri — DISMISSED** [review]

**Comment by Progi1984** [comment]
@nesrineabdmouleh Needs a rebase

**Review by boubkerbribri — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**